### PR TITLE
Added Swag Opportunity for NativeScript

### PR DESCRIPTION
**Pull request by BandhiyaHardik**
Added Swag Opportunity for NativeScript

- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

This fixes https://github.com/swapagarwal/swag-for-dev/issues/1184

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -647,6 +647,15 @@
     "dateAdded": "2019-01-25T06:45:29.000Z",
     "tags": ["expired", "stickers"]
   },
+  {
+    "name": "NativeScript",
+    "difficulty": "medium",
+    "description": "Make your mark on Native Script! Either build a neat project on StackBlitz, or contribute a quality PR. Tweet about your work and mention @NativeScript. Be sure to DM us your address to claim your NativeScript Stickers!!",
```
Comment (aslafy-z) [inline data.json:653]:
```suggestion
    "description": "Make your mark on Native Script! Either build a neat project on StackBlitz, or contribute a quality PR. Tweet about your work and mention @NativeScript. DM maintainers to claim your stickers!",
```

File: data.json [hunk 2]
```diff
@@ -647,6 +647,15 @@
...
+    "reference": "https://twitter.com/NativeScript/status/1709066192605061392",
+    "image": "https://i.imgur.com/jrwefsD.jpg",
```
Comment (aslafy-z) [inline data.json:655]: Can you please choose another image so it renders ok in the preview ? 🙏

Comment (BandhiyaHardik) [reply data.json:655]: I will do it now

**Comment by aslafy-z** [comment]
Thank you @BandhiyaHardik

**Pull request merged**